app/controllers/assignment_invitations_controller.rb

```diff
@@ -54,7 +54,7 @@ def create_repo
         GitHubClassroom.statsd.increment("v2_exercise_repo.import.retry")
       end
       current_invitation_status.waiting!
-      AssignmentRepo::CreateGitHubRepositoryJob.perform_later(current_assignment, current_user)
+      AssignmentRepo::CreateGitHubRepositoryJob.perform_later(current_assignment, current_user, 3)
```

Comment:
Reading this method invocation, clear that `current_assignment` is providing the assignment to the method, and `current_user` is providing the user, but it's difficult or impossible to infer the purpose of `3` without looking at the method definition. Could you make `retries` a kwarg in this method?